### PR TITLE
[TASK] Add compatibility with TYPO3 v12

### PR DESCRIPTION
**Pull request by b13-michaelsemle**

### CODE REVIEW

**Review by ochorocho — APPROVED** [review]
LGTM

File: composer.json [hunk 1]
```diff
@@ -18,18 +18,24 @@
   "require": {
     "php": "^8.1",
     "typo3/cms-form": "^11.5 || ^12.3",
-    "typo3/cms-core": "^11.5 || ^12.3",
     "typo3/cms-frontend": "^11.5 || ^12.3",
     "typo3/cms-install": "^11.5 || ^12.3",
-    "typo3/cms-filelist": "^11.5 || ^12.3"
+    "typo3/cms-filelist": "^11.5 || ^12.3",
+    "typo3/cms-core": "^12.4"
```
Comment (b13-michaelsemle) [inline composer.json]: Now you are using typo3/cms-core ^12 only in composer json. That should be 11 and 12 right?

**Comment by ervaude** [comment]
Hey friends, can we merge this an tag a 3.0?

**Review by bmack — APPROVED** [review]

**Review by bmack — APPROVED** [review]

**Comment by ervaude** [comment]
great, @ochorocho can you have alook at the failing tests?

**Review by ochorocho — APPROVED** [review]
👍 Extended/updated the tests to work with v11 and v12

**Review by ochorocho — APPROVED** [review]

**Pull request merged**